### DISCUSSION TITLE
All hardware settings on options-menu on webpage.

### DISCUSSION THREAD

**Discussion by HansNetcom** [serkri/SmartEVSE-3#140] [Ideas]
Would it be possible / make sense to make all settings on the hardware module accessible and changeable on the website page behind a settings options?
For me the Actions: Reboot, Update and Raw Data could also be removed from the main-page and be part of the settings menu.

**Comment by dingo35** [comment]
Be my guest, I like to stay away from optics but can appreciate nice menu's/tabs erc.

**Comment by HansNetcom** [comment]
Unfortunately I do not have the programming skills to build it..sorry
